pkg/sdk/row_access_policies_gen.go

```diff
@@ -110,3 +111,21 @@ type RowAccessPolicyDescription struct {
 	ReturnType string
 	Body       string
 }
+
+// TODO(SNOW-1596962): Fully support VECTOR data type
+func (d *RowAccessPolicyDescription) Arguments() []map[string]any {
```

Comment:
for the logic like this, it would be great to add an integration test with different arguments combinations (different data types, especially with number parameters) and assert what is returned in signature (that way we will know when Snowflake changes something)

Reply:
Done.